lib/core/util.c

```diff
@@ -944,4 +944,7 @@ void h2o_cleanup_thread(void)
     h2o_mem_clear_recycle(&h2o_mem_pool_allocator);
     h2o_mem_clear_recycle(&h2o_http2_wbuf_buffer_prototype.allocator);
     h2o_mem_clear_recycle(&h2o_socket_buffer_prototype.allocator);
+
+    extern __thread h2o_buffer_prototype_t h2o__http2client_wbuf_buffer_prototype;
```

Comment:
Could you please move the declaration to httpclient.h? I think we have a consistent pattern in declaring global variables in header files.

Reply:
Yes, will fix it later.